etc/format-scripts/check-file.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
```

Comment:
Make sure `$1` is set
```suggestion
set -eu
```